store/sdk/src/main/java/org/apache/carbondata/sdk/file/CarbonWriterBuilder.java

```diff
@@ -284,6 +286,28 @@ public CarbonWriterBuilder withBlockSize(int blockSize) {
     return this;
   }
 
+  /**
+   * @param localDictionaryThreshold is localDictionaryThreshold,default is 1000
+   * @return
+   */
+  public CarbonWriterBuilder localDictionaryThreshold(int localDictionaryThreshold) {
+    if (localDictionaryThreshold <= 0) {
+      throw new IllegalArgumentException("blockSize should be between greater than 0");
+    }
+    this.localDictionaryThreshold = localDictionaryThreshold;
+    return this;
+  }
+
+  /**
+   * @param isLocalDictionaryEnabled enable local dictionary  , default is false
+   * @return updated CarbonWriterBuilder
+   */
+  public CarbonWriterBuilder isLocalDictionaryEnabled(boolean isLocalDictionaryEnabled) {
```

Comment:
change the method name to enableLocalDictionary

Reply:
Done